docs/checklist_applicationclass_2-markdown_v1.1.md

```diff
@@ -0,0 +1,157 @@
+
+* Is there currently no need for improvement and is the recommendation addressed appropriately? Status: **ok** 
+* Is there any potential for improvement that should be implemented in the near future? Status: **todo**, record the need for improvement 
+* Is the recommendation not yet relevant but could be helpful in a later development phase? Status: **future** 
+* Is the recommendation not meaningfully implementable within the development context? Status: **n.a.** (not applicable) explain the reason 
+
+In case of questions, you can contact the Software Engineering Contact of your institute or facility.
+
+> Please note the status between "[]" and list remarks below a recommendation.
+
+-->
+
+## Summary of Results
+The software CosmoScout VR implements 42 recommendations of application class 2. 
+
+The focus of future improvements is on improving the testing capabilities.
+
+## Table of Contents
+[[Qualification](#qualifizierung)] [[Requirements Management](#anforderungsmanagement)] [[Software Architecture](#software-architektur)] [[Change Management](#aenderungsmanagement)] [[Design and Implementation](#design-implementierung)] [[Software Test](#software-test)] [[Release Management](#release-management)] [[Automation and Dependency Management](#automatisierung-abhaengigkeiten)] 
+
+## Qualification <a name="qualifizierung"></a>
+**EQA.1** - **[ok]** The software responsible recognises the different application classes and knows which is to be used for his/her software. *(from application class 1)*
+
+**EQA.2** - **[ok]** The software responsible knows how to request specific support at the beginning and during development as well as to exchange ideas with other colleagues on the subject of software development. *(from application class 1)*
+
+**EQA.3** - **[ok]** The persons involved in the development determine the skills needed with regard to their role and the intended application class. They communicate these needs to the supervisor. *(from application class 1)*
+
+**EQA.4** - **[ok]** The persons involved in the development are given the tools needed for their tasks and are trained in their use. *(from application class 1)*
+
+## Requirements Management <a name="anforderungsmanagement"></a>
+
+:information_source: _For now, all of these are marked `[n.a.]` as this is usually done on a per-project basis in which CosmoScout VR is applied._ 
+
+**EAM.1** - **[n.a.]** The problem definition is coordinated with all parties involved and documented. It describes the objectives, the purpose of the software, the essential requirements and the desired application class in a concise, understandable way. *(from application class 1)*
+
+**EAM.2** - **[n.a.]** Functional requirements are documented at least including a unique identifier, a description, the priority, the origin and the contact person. *(from application class 2)*
```

Comment:
We should mark this as future work and include at least the basic requirements as a list and mention partners

Reply:
```suggestion
**EAM.2** - **[todo]** Functional requirements are documented at least including a unique identifier, a description, the priority, the origin and the contact person. *(from application class 2)*
```